php-base/stackdriver-files/enable_stackdriver_prepend.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+$options = getopt('a:o:') + [
+    'a' => '/app',
+    'o' => 'php://stdout'
+];
+
+$autoload = $options['a'] . '/vendor/autoload.php';
```

Comment:
I think we can ignore this for now. The way it currently works will also fail with a non-standard `vendor-dir` because we have hard-coded fallback paths under `/app/vendor/...`.

It looks like we can query for the vendor directory with `composer config vendor-dir`, but we can do this in a separate PR.